src/emmc.c

```diff
@@ -270,10 +270,10 @@ pu_emmc_write_data(PuFlash *flash,
 
             if (g_regex_match_simple(".tar", path, G_REGEX_CASELESS, 0)) {
                 g_debug("Extracting '%s' to '%s'", path, part_mount);
-                pu_mount(part_path, part_mount);
+                pu_mount(part_path, part_mount, error);
```

Comment:
We must check the return value of `pu_mount()` in order to prevent piling up errors and exit appropriately.